packages/react/src/modules/slack/hooks/useSlackChannels.ts

```diff
@@ -46,13 +42,22 @@ export function useSlackChannels(
               knockChannelId: knockSlackChannelId,
             })
             .then((res) => {
-              const { channels } = res;
-              setData({ channels });
+              if (res.code === "ERR_BAD_REQUEST") {
+                const message = res.response?.data?.message
```

Comment:
Should we have a default error message in case there isn't one in the response? Since we're checking on the presence of the error message, I think we should.

Reply:
Yes - i'll come back to this one actually though when I spruce up the channel picker, there will be more to do on error handling there anyway